dao/mongodb.rya/src/main/java/org/apache/rya/mongodb/document/util/DocumentVisibilityUtil.java

```diff
@@ -328,4 +328,22 @@ public static boolean doesUserHaveDocumentAccess(final Authorizations authorizat
         }
         return list.toArray(new Object[0]);
     }
+
+    /**
+     * Converts a {@link List} into an array of {@link Object}s.
```

Comment:
It might be worth mentioning that it flattens lists that are found within inputList into the returned array. What does this do to other data structures though? You'll leave a Set or Map as such in the array? Are other collections just allowed to be in the returned array?

Reply:
Removed this method anyway, turned out to be unnecessary. (I had some incorrect way of converting a Document to a DBObject; if done properly, the existing methods are sufficient.)